ts/Series/Treegraph/TreegraphSeries.ts

```diff
@@ -0,0 +1,935 @@
+     *               lineWidth, negativeColor, nodes, sortIndex, zoneAxis,
+     *               zones
+     *
+     * @since        next
+     * @product      highcharts
+     * @requires     modules/treemap.js
+     * @requires     modules/treegraph.js
+     * @optionparent plotOptions.treegraph
+     */
+    public static defaultOptions: TreegraphSeriesOptions = merge(
+        TreemapSeries.defaultOptions,
+        {
+            /**
+             * Flips the positions of the nodes of a treegraph along the
+             * horizontal axis (vertical if chart is inverted).
+             *
+             * @type {boolean}
+             * @since next
+             * @default false
+             * @product highcharts
+             *
+             * @sample highcharts/series-treegraph/reversed-nodes
+             *         Treegraph series with reversed nodes.
+             */
+            reversed: false,
+            /**
+             * @extends   plotOptions.series.marker
+             * @excluding enabled, enabledThreshold
+             */
+            marker: {
+                radius: 10,
+                lineWidth: 0,
+                symbol: 'circle',
+                fillOpacity: 1,
+                states: {}
+            },
+            link: {
+                /**
+                 * The color of the links between nodes.
+                 *
+                 * @type {Highcharts.ColorString}
+                 * @private
+                 */
+                color: Palette.neutralColor60,
+                /**
+                 * The line width of the links connecting nodes, in pixels.
+                 * @type {number}
+                 *
+                 * @private
+                 */
+                lineWidth: 1,
+                /**
+                 * Radius for the rounded corners of the links between nodes.
+                 * Works for `default` link type.
+                 *
+                 * @private
+                 */
+                radius: 10,
+                cursor: 'default',
+                /**
+                 * Type of the link shape.
+                 *
+                 * @sample   highcharts/series-treegraph/link-types
+                 *           Different link types
+                 *
+                 * @type {'default' | 'curved' | 'straight'}
+                 * @product highcharts
+                 *
+                 */
+                type: 'curved'
+                /**
+                 * Modifier of the shape of the curved link. Works best for
+                 * values between 0 and 1, where 0 is a straight line, and 1 is
+                 * a shape close to the default one.
+                 *
+                 * @default 0.5
+                 * @type {number}
+                 * @since next
+                 * @product highcharts
+                 * @apioption series.treegraph.link.offset
+                 *
+                 */
```

Comment:
Why is this option called `offset`? Judging from the description, it sounds similar to the Sankey [curveFactor](https://api.highcharts.com/highcharts/series.sankey.curveFactor). If it is indeed the same, it should have the same name.